bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/VerifyDigestFilesService.scala

```diff
@@ -49,15 +43,13 @@ class VerifyDigestFilesService(
     bagLocation: BagLocation,
     digestFiles: Seq[BagDigestFile]
   )(implicit materializer: Materializer): Future[BagVerification] = {
-    val verificationStart = Instant.now
-
+    val bagVerification = BagVerification(startTime = Instant.now)
```

Comment:
Perhaps we could make `Instant.now` then default value for this parameter?

Reply:
I think we discussed this and decided that it's better to be explicit when creating a verification what the start time is.